### PR TITLE
Add solution to html-forms project

### PR DESCRIPTION
**Pull request by elPetit69**
```
This is a PR template, If you are adding a solution link to the curriculum leave this as is. If you are not, then delete it and write the message you wish.
Thank you,
The Odin Project team.

```

### CODE REVIEW

File: html_css/project_html_forms.md [hunk 1]
```diff
@@ -51,7 +55,7 @@ Send us your solution so we can show others! Submit a link to the Github repo wi
 * [Dominik Stodolny's solution](https://github.com/dstodolny/mint) - [View in browser](https://htmlpreview.github.io/?https://github.com/dstodolny/mint/blob/master/index.html)
 * [AtActionPark's solution](https://github.com/AtActionPark/odin_html_forms) - [View in browser](https://htmlpreview.github.io/?https://github.com/AtActionPark/odin_html_forms/blob/master/main.html)
 * [Eleanor's solution (with Flexbox)](https://github.com/mixophrygian/mintForm) - [View in browser](https://htmlpreview.github.io/?https://github.com/mixophrygian/mintForm/blob/master/index.html)
-* [Joseph McConnell's solution](https://github.com/JJMcConnell/TheOdinProject/tree/master/HTML%20Forms%20practice%20(Mint%20clone)) - [View in browser](https://htmlpreview.github.io/?https://github.com/JJMcConnell/TheOdinProject/blob/master/HTML%20Forms%20practice%20(Mint%20clone)/Mmmmmity.html)
+* [Joseph McConnell's solution](<https://github.com/JJMcConnell/TheOdinProject/tree/master/HTML%20Forms%20practice%20(Mint%20clone)>) - [View in browser](<https://htmlpreview.github.io/?https://github.com/JJMcConnell/TheOdinProject/blob/master/HTML%20Forms%20practice%20(Mint%20clone)/Mmmmmity.html>)
```
Comment (105ron) [inline html_css/project_html_forms.md]: Why did this solution get modified?

Comment (elPetit69) [reply html_css/project_html_forms.md]: Hi! That's the odd part. I didn't change it, just added a line with my solution.  I think I've messed something with pulling/ merging repo. I tried to revert the commit and make it one more time again but it doesn't look that it worked. Sorry, I'm new to this.

Comment (105ron) [reply html_css/project_html_forms.md]: That's a bit odd. A do remember someone formatting their solution like that previously. I'm not sure how it's come back in this PR though. Maybe you could just add it via the website to create a new pull request? Or sync your fork again to try and fix it and make a new PR? I can just close this if you can fix it up in a different PR. It's no issue to make a few PR's to learn more about Git. :smile:

Comment (elPetit69) [reply html_css/project_html_forms.md]: Ok, I'll do that. Thanks :)

**Pull request closed without merging**